02Lesson/01Project/index.html

```diff
@@ -1,9 +1,16 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>Everyday Pest Control</title>
+  </head>
+  <body>
+    <img src="http://2.bp.blogspot.com/-IYs14fZjSho/UktAf129fPI/AAAAAAAAA4Q/zLG5r14wY_U/s1600/exterminador+cucarachas.jpg"></img>
```

Comment:
`img` tag is self closing and does not need a closing tag. `alt` attribute is missing.